lib/blkproto.ml

```diff
@@ -260,6 +260,7 @@ module Req = struct
   | Flush -> "Flush" | Op_reserved_1 -> "Op_reserved_1" | Trim -> "Trim"
   | Indirect_op -> "Indirect_op"
 
+  [@@@warning "-38"]
   exception Unknown_request_type of int
```

Comment:
Just delete it? Also, I think `@@@` applies to everything following, not just this item, which from the formatting I'm guessing isn't what you intended.